app/Http/Controllers/Api/V1/TeamController.php

```diff
@@ -147,21 +152,47 @@ public function store(TeamRequest $teamRequest)
     {
         $data = $teamRequest->validated();
```

Comment:
Why we are having complete logic in controller? As we are implementing repository pattern.